op-batcher/batcher/driver.go

```diff
@@ -294,7 +296,14 @@ func (l *BatchSubmitter) loop() {
 	for {
 		select {
 		case <-ticker.C:
-			l.loadBlocksIntoState(l.shutdownCtx)
+			if err := l.loadBlocksIntoState(l.shutdownCtx); errors.Is(err, ErrReorg) {
+				err := l.state.Close()
+				if err != nil {
+					l.log.Error("error closing the channel manager to handle a L2 reorg", "err", err)
+				}
+				l.publishStateToL1(queue, receiptsCh, true)
+				l.state.Clear()
+			}
```

Comment:
Does it matter that we're going to fall step out of this error handling and call publishStateToL1 again below? I expect it winds up being a no-op but it feels a bit weird to have just reset everything and then carry on like everything's normal.

Reply:
I added a `continue` statement to escape the case stmt & continue with the select loop after the error case is handled.

Reply:
I think it's a no-op, but definitely weird.